docs/aiops-projects.md

```diff
@@ -0,0 +1,41 @@
+
+13.  **Insights Invocation Hints** : Given a timestamp of insight archive uploads, deduce how systems checkin and how they were initially registered. 
+
+14.  **Insights Drift Analysis Baselines** : Drift Analysis application enables users to compare system configuration of one system to other systems or baselines in the cloud management services inventory. Baselines are configurations (set of name / value facts) that can be defined from scratch, as a copy of an existing system configuration, or as a copy of an existing baseline. We propose a method to recommend baseline configurations automatically to the users. They can then use the recommendation or tweak it further to compare their systems with the baselines. This approach of recommending baselines by utilizing the knowledge of other systems in the account would save time for users by identifying and recommending baselines. This would assist in standardizing RHEL configurations and in improving RHEL configuration management and operations.
+
+15.  **Insights SAP Analysis** : In this project, we analyze SAP instances on user systems through data collected from Insights. We create a superset dashboard that illustrates the topology of SAP workloads running on RHEL systems.
+
+16. **CJA Topic Modeling** : The Voice of Customer team has been focused on developing a process to systematically track customer sentiment for a larger portfolio of our customers and products in order to know our customers better. The VoC team collaborated with AICoE to scale this process by using AI techniques to gather meaningful insights from customer data. In order to analyze customer feedback for themes we used topic analysis techniques and were able to detect key themes and topics in customer verbatim that need attention. By tying the key themes and topics to customer satisfaction scores and customer sentiment values, we can get a fair understanding of customer’s pain points and help one better improve one’s products and services. 
+
+17. **SRE** : The goal of this project is to outline and practice a common ‘bleeding edge technology’ approach for all AI CoE endeavors to operate their software components and services. All services target OpenShift as the primary platform, but still extending to the full stack - from underlying hardware up to the applications running on top of a service. E.g. supporting custom AI service on Seldon provided by DataHub on PSI on OpenStack.
```

Comment:
Does this have another name, like AI Ops SRE? I think the existing title if kind of vague.

Reply:
@hemajv What do you suggest?